package.json

```diff
@@ -33,15 +36,21 @@
         "eslint-plugin-prettier": "^4.2.1",
         "eslint-plugin-vitest": "^0.0.48",
         "husky": "^8.0.0",
+        "mockdate": "^3.0.5",
         "prettier": "^2.8.4",
         "sst": "2.24.12",
-        "typescript": "^4.9.5"
+        "typescript": "^4.9.5",
+        "vitest": "^0.29.1"
     },
     "workspaces": [
         "packages/**/*"
     ],
     "dependencies": {
-        "@aws-sdk/client-rds-data": "^3.282.0",
+        "@aws-sdk/client-rds-data": "^3.418.0",
+        "@aws-sdk/client-s3": "^3.418.0",
+        "@aws-sdk/client-sns": "^3.418.0",
+        "@aws-sdk/types": "^3.418.0",
```

Comment:
In fact, these types can be dev deps too

Reply:
done